src/components/redirects/redirects-routes.ts

```diff
@@ -6,9 +6,9 @@ import { requiresAuthMiddleware } from "../../middleware/requires-auth-middlewar
 const router = express.Router();
 
 router.get(
-  PATH_DATA.MANAGE_YOUR_ACCOUNT.url,
+  [PATH_DATA.MANAGE_YOUR_ACCOUNT.url, PATH_DATA.SETTINGS.url],
```

Comment:
Huh, I didn't know this could take an array.

So what are we saying here that if you try to go to either the URLs for Manage your account or settings...
then redirect to settings?

Are we deprecating both? I need to double check the design again...

Cool to know .get() [takes an array](https://expressjs.com/en/api.html#app.get) though, TIL.

Reply:
ah wait, I get this now, because we already had the old Manage your account URL and now we're also deprecating settings, right?

Sorry up to speed now.

Reply:
Yeah that's right, "Settings" used to be "Manage your account", so there was already a redirect in place for that. But now "Settings" is becoming "Security", so we need to ensure both `/manage-your-account` and `/settings` redirect the user to `/security` now.